rakefile.rb

```diff
@@ -31,6 +31,18 @@
 CLEAN.include(FileList["src/**/#{CONFIGURATION}"])
 CLEAN.include(FileList["test/**/#{CONFIGURATION}"])
 
+desc "Restore NuGet packages"
+exec :nuget_restore do |cmd|
+   cmd.command = "tools/nuget/NuGet.exe"

```

Comment:
Nope, that does not exist. It has to be `dependencies/Nancy/tools/nuget/NuGet.exe`